apps/mobile/src/app/components/ModalBottom/ModalBottom.tsx

```diff
@@ -16,22 +16,31 @@ interface ModalBottomProps {
 }
 
 const ModalBottom = (props: ModalBottomProps) => {
-  const { bottomSheet, auction, confirmModal, bid } = props;
-  const [enteredValue, setEnteredValue] = useState(`${props.bid + 1}`);
+  const { bottomSheet, auction, confirmModal } = props;
+
+  let bid = props.bid;
+  const maxPrice = 99999;
```

Comment:
Can You fetch this value form environment configuration? I mean MAX_BID_VALUE from .env :)